Aztec/Classes/TextKit/TextView.swift

```diff
@@ -148,6 +148,17 @@ open class TextView: UITextView {
     let defaultFont: UIFont
     var defaultMissingImage: UIImage
 
+    // MARK: - Properties: Processors
+
+    /// These processors will be executed on any HTML you provide to the method `setHTML()` and
+    /// before Aztec attempts to parse it.
+    ///
+    public var preProcessors: PipelineProcessor?
+
+    /// These processors will be executed right before returning the HTML in `getHTML()`.
+    ///
+    public var postProcessors: PipelineProcessor?
```

Comment:
The plural name sucks tbh.  I only realized after submitting the PR and was a bit too lazy to change it, heh.

I'll change it though since the observation makes sense.

A. The downside of doing that is that the pretty-modular logic in `PipelineProcessor` would move to `TextView`.  I'd rather avoid that.

B. Sounds more like it.